.github/workflows/pr.yml

```diff
@@ -31,19 +32,21 @@ jobs:
       - name: Collect Diagnostics
         if: always()
         run: ./scripts/collect-diagnostics.main.kts
-      - uses: actions/upload-artifact@v2.2.4
+      - uses: actions/upload-artifact@v3
         if: always()
         with:
           name: tests-gradle.zip
           path: diagnostics.zip
+
   tests-no-gradle:
     runs-on: macos-11
     steps:
```

Comment:
And if we should run CI on ubuntu? It should be faster.

Reply:
I wish we could do that. Unfortunatly, some of the Gradle tests run multiplatform projects that include Apple targets. We could split them but I don't think it's worth the trouble for now.